README.md

```diff
@@ -109,6 +109,7 @@ The following table lists all valid connection properties:
 | Connection Property        | Required | Comment                                                                       |
 |----------------------------|----------|-------------------------------------------------------------------------------|
 | ACCOUNT                    | Yes      | Your full account name might include additional segments that identify the region and cloud platform where your account is hosted	|
+| APPLICATION                | No       | **_Snowflake partner use only_**: Specifies the name of a partner application to connect through .NET. The name must match the following pattern:  ^\[A-Za-z](\[A-Za-z0-9.-]){1,50}$ (one letter followed by 1 to 50 letter, digit, .,- or, \_ characters).   |
```

Comment:
@sfc-gh-dyoshinaga This is the first time I have seen a specific property be flagged as partner only. The partner-specific docs I have worked on are all PrPr. I assume this is ok.

Reply:
This is actually a property that's been available in other drivers and connectors for some time now. We're just adding this to the .NET driver now.

e.g. the property is already supported in the ODBC driver:

https://docs.snowflake.com/en/user-guide/odbc-parameters.html#:~:text=the%20connection%20parameters.-,[application](https://docs.snowflake.com/en/user-guide/odbc-parameters.html#:~:text=the%20connection%20parameters.-,application,-Snowflake%20partner%20use),-Snowflake%20partner%20use

the JDBC driver:

https://docs.snowflake.com/en/user-guide/jdbc-parameters.html#application

Reply:
Great. Thanks Doug!